src/stream.h

```diff
@@ -98,8 +98,11 @@ struct client;
 
 /* Flags for streamLookupConsumer */
 #define SLC_NONE      0
-#define SLC_NOCREAT   (1<<0) /* Do not create the consumer if it doesn't exist */
-#define SLC_NOREFRESH (1<<1) /* Do not update consumer's seen-time */
+#define SLC_REFRESH   (1<<0) /* Do update consumer's seen-time */
+
+/* Flags for streamCreateConsumer */
+#define SCC_NONE      0
+#define SCC_JUSTCREATE (1<<0) /* Do not notify key space and dirty++ if consumer created */
```

Comment:
it's a bit odd that for one function the default flag do just the minimum and the additional flags specify to do extra work, and in the other function the default flag is to do everything and the additional flags tell it to avoid the optional parts.

i'd vote to change that.
maybe?
```c
/* Flags for streamLookupConsumer */
#define SLC_DEFAULT      0
#define SLC_NO_REFRESH   (1<<0) /* Do not update consumer's seen-time */

/* Flags for streamCreateConsumer */
#define SCC_DEFAULT      0
#define SCC_NO_NOTIFY  (1<<0)
#define SCC_NO_DIRTIFY (1<<1)
```

Reply:
good change!